Formula/hz.rb

```diff
@@ -0,0 +1,27 @@
+class Hz < Formula
+  desc "Golang HTTP framework for microservices"
+  homepage "https://github.com/cloudwego/hertz"
+  url "https://github.com/cloudwego/hertz/archive/refs/tags/v0.6.1.tar.gz"
+  sha256 "2fc3d721efa23e12b3f7a168b3d54d0b4301ccf34fd1cdc63b8c6b30a1a0c98b"
+  license "Apache-2.0"
+  head "https://github.com/cloudwego/hertz.git", branch: "develop"
+
+  depends_on "go" => :build
+
+  def install
+    cd "cmd/hz" do
+      system "go", "build", *std_go_args(ldflags: "-s -w")
+    end
```

Comment:
Does this work?
```suggestion
    system "go", "build", *std_go_args(ldflags: "-s -w"), "./cmd/hz"
```
If you apply this change, please squash it into your existing commit (instead of creating a new one), and use the commit message
```
hz 0.6.1 (new formula)
```

Reply:
yes, it works, as there is a separate go.mod file in cmd/hz subfolder

Reply:
Ok, let's go with that suggestion then. It's easier to read and takes up less space.

Reply:
sorry, I mistake your means...

``` 
system "go", "build", *std_go_args(ldflags: "-s -w"), "./cmd/hz"
```
does not work as 
```
main module (github.com/cloudwego/hertz) does not contain package github.com/cloudwego/hertz/cmd/hz
```

Reply:
I see, ok, never mind then.